diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -965,7 +965,12 @@ int DifferentialFunctionComparator::cmpGlobalValues(GlobalValue *L,
                 }
             }
             return 0;
-        } else
+        } else if (NameL != NameR && GVarL && GVarR && !GVarL->hasInitializer()
+                   && !GVarR->hasInitializer()) {
```

Comment:
I think that we should also add `&& GVarL->isConstant() && GVarR->isConstant`. Preferably add this before checking that the variables have initializers.